ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/cv-pathfinder-pathfinder.yml

```diff
@@ -284,9 +284,16 @@ router_path_selection:
   - name: Equinix
     id: 103
     ipsec_profile: CP-PROFILE
+    keepalive:
+      interval: 100
+      failure_threshold: 12
   - name: Satellite
     id: 104
     ipsec_profile: CP-PROFILE
+    keepalive:
+      auto: true
+      interval: 100
+      failure_threshold: 15
```

Comment:
The auto vs intervals are mutually exclusive. `eos_designs` should only produce one of them.